### ISSUE TITLE
Formatted item pastes as plain text

### ISSUE THREAD

**Issue by paul1149** [hluk/CopyQ#985]
Hello again,

I have the same problem as in this thread: https://github.com/hluk/CopyQ/issues/768. I have an item that has some basic text formatting that never shows up. I've tried pasting into the ebay listing description, and into LibreOffice Writer. And when I hit F7 within CopyQ, to preview the item, it too shows no formatting.

In that thread, it was suggested to look at this setting:

https://user-images.githubusercontent.com/38054/29611580-4f69d912-87fe-11e7-8ec3-1a4356c1862c.png

But when I go there the list in completely empty! I'm not sure how to find and manage plugins, which this seems to be related to.

I'm on CopyQ 3.5, on MX Linux 17.1 x64. Thanks for any direction you can give.

BW,

**Comment by hluk** [comment]
Looks like no plugins were found, you can check log if it has any warnings or errors about it.

Is it possible that there is separate package with plugins?

**Comment by paul1149** [comment]
Yes, that was the case, there is a separate plugins package, which I have now installed. I closed and reopened CopyQ, and reordered the Items hierarchy to have TEXT on the top, as that other thread suggested. Now I see the markup in the Preview Pane, but still nothing is formatted when I paste into LibreOffice.

Ah, I see the problem. Pasting from the CQ interface works perfectly. What doesn't carry the formatting forward is using the Snippets command addon, which is what I always use to paste this text. Can that be remedied? Thanks!
